cpp/test/CMakeLists.txt

```diff
@@ -6,13 +6,13 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(googletest)
 # run google test
-enable_testing()
+
 
 set(TEST_BINARY ${CMAKE_PROJECT_NAME}_test)
 file(GLOB_RECURSE TEST_SOURCES LIST_DIRECTORIES false *.cc *.hh)
```

Comment:
Can we copy test/resources dir to build dir to avoid the resource dir with relative path in the following code?